packages/frontend/src/routes/app/posts/[id]/page.tsx

```diff
@@ -141,6 +142,7 @@ const PostDetailsPage: React.FC = () => {
                         votedNonce={post.votedNonce}
                         votedEpoch={post.votedEpoch}
                         onVote={handleVote}
+                        epoch={post.epoch}
```

Comment:
Can you adjust the params order? Make is consistent with the Post component :)

Reply:
Sorry that I forgot to push the latest commit in local. Already fixed it. Thanks.